ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -596,7 +596,8 @@ objects:
         elif [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
             # HACK: try to "poke" the token endpoint before the test starts
             for i in $(seq 1 30); do
-              code="$( curl -s -o /dev/null -w "%{http_code}" https://oauth2.googleapis.com/token -X POST -d '' )"
+              echo "check Google's OAuth API ($i)"
```

Comment:
Not really useful - why don’t you put the curl in an if and print the rc on fail?

if ! code=$( ... ); then
  echo “failed with rc=$?”
if

Would be much more useful

Reply:
>    echo “failed with rc=$?”

This doesn't work in my local testing:

```console
$ echo $BASH_VERSION
4.2.46(2)-release
$ if ! code="$( curl -s -o /dev/null -w "%{http_code}" https://does-not-exist.example.com -X POST -d '')"; then echo "Failed to POST https://oauth2.googleapis.com/token with $?" 1>&2; fi
Failed to POST https://oauth2.googleapis.com/token with 0
```

and it doesn't seem like it's worth temporarily turning off `set -e`.  But pushed something like this with 4759620e0 -> 3ef39054f.

Reply:
thought of a better way...

Reply:
Rerolled again with 3ef39054f -> d612a99f1, now I log the exit code.